pulumi-programs/ttl-stacks/app/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM --platform=linux/amd64 public.ecr.aws/lambda/nodejs:16 as builder
+WORKDIR /usr/app
+COPY package.json index.ts  ./
+RUN npm install
+RUN npm run build
+
+
+FROM --platform=linux/amd64 public.ecr.aws/lambda/nodejs:16
+
+ENV AWS_REGION="${AWS_REGION}"
+ENV AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}"
+ENV AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}"
+ENV AWS_SESSION_TOKEN="${AWS_SESSION_TOKEN}"
```

Comment:
why are all of these explicit env declarations required?

Reply:
Does this set the runtime value to the build time value? If so - that isn't what we want to do. When the lambda executes it is launched with session-scoped credentials provided by the lambda runtime. We should use those as we don't have to worry about the credentials we pass into the lambda expiring.